ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -145,6 +145,10 @@ http://www.freedesktop.org/software/systemd/man/systemd.exec.html
 valid for [Service], [Socket], [Mount], or [Swap] sections
 }
 type ${project.artifactId}_unitfile_config_systemd_exec = {
+    'CacheDirectoryMode' ? long(1..1023) # octal notation
```

Comment:
How do I specify `0755` ? Is the intention it should be written `755` ? Is that consistent with other components?

Reply:
Why wouldn't you be able to write `0755`? 
I'm not that deep into PAN but 755 is decimal, 0755 is octal?